README.md

```diff
@@ -1,3 +1,17 @@
+- [Xunit.Gherkin.Quick](#xunitgherkinquick)
+  - [Project Sponsors](#project-sponsors)
+  - [Getting Started](#getting-started)
+      - [Xunit Test Project](#xunit-test-project)
+      - [Install Nuget Package](#install-nuget-package)
+      - [Create Gherkin Feature File](#create-gherkin-feature-file)
+      - [Implement Feature Scenario](#implement-feature-scenario)
+      - [Run Scenario](#run-scenario)
+      - [Add More Scenarios](#add-more-scenarios)
+  - [Got Stuck?](#got-stuck)
+  - [Special Thanks](#special-thanks)
+  - [Documentation and Reference](#documentation-and-reference)
+  - [See Also](#see-also)
+
```

Comment:
I like this table of contents for the page, but I am not sure if it will help starters being located before the Xunit.Gherkin.Quick intro text. If you put it after that intro (before project sponsors), does it still work fine? If so, that may be a better place for it.
When starters come to the project, I think it helps if they see the intro text first since it lists framerowk's advantages.

Reply:
It is autogenerated with this [Visualstudio-Code Extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#table-of-contents)